browsers/firefox_android.json

```diff
@@ -54,237 +54,237 @@
         "38": {
           "release_date": "2015-05-12",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/38",
+          "release_notes": "https://www.mozilla.org/firefox/android/38.0beta/releasenotes/",
           "status": "retired"
         },
         "39": {
           "release_date": "2015-07-02",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/39",
+          "release_notes": "https://www.mozilla.org/firefox/android/39.0beta/releasenotes/",
           "status": "retired"
         },
         "40": {
           "release_date": "2015-08-11",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/40",
+          "release_notes": "https://www.mozilla.org/firefox/android/40.0beta/releasenotes/",
           "status": "retired"
         },
         "41": {
           "release_date": "2015-09-22",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/41",
+          "release_notes": "https://www.mozilla.org/firefox/android/41.0beta/releasenotes/",
           "status": "retired"
         },
         "42": {
           "release_date": "2015-11-03",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/42",
+          "release_notes": "https://www.mozilla.org/firefox/android/42.0beta/releasenotes/",
           "status": "retired"
         },
         "43": {
           "release_date": "2015-12-15",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/43",
+          "release_notes": "https://www.mozilla.org/firefox/android/43.0beta/releasenotes/",
           "status": "retired"
         },
         "44": {
           "release_date": "2016-01-26",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/44",
+          "release_notes": "https://www.mozilla.org/firefox/android/44.0beta/releasenotes/",
           "status": "retired"
         },
         "45": {
           "release_date": "2016-03-08",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/45",
+          "release_notes": "https://www.mozilla.org/firefox/android/45.0beta/releasenotes/",
           "status": "retired"
         },
         "46": {
           "release_date": "2016-04-26",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/46",
+          "release_notes": "https://www.mozilla.org/firefox/android/46.0beta/releasenotes/",
           "status": "retired"
         },
         "47": {
           "release_date": "2016-06-07",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/47",
+          "release_notes": "https://www.mozilla.org/firefox/android/47.0beta/releasenotes/",
           "status": "retired"
         },
         "48": {
           "release_date": "2016-08-02",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/48",
+          "release_notes": "https://www.mozilla.org/firefox/android/48.0beta/releasenotes/",
```

Comment:
This shoudn’t point to the beta release notes, but to the [normal release notes](https://www.mozilla.org/firefox/android/48.0/releasenotes/).